tests/core/test_config.py

```diff
@@ -0,0 +1,7 @@
+from dataall.core.config import config
+
+
+def test_config():
+    """Checks that properties are read correctly"""
+    modules = config.get_property("modules")
+    assert "notebooks" in modules
```

Comment:
[Question] - What happens when notebooks is not added to the modules? Is this test going to be particular for each implementation

Reply:
This test is needed to make sure that we don't miss any module. By default, we want to include all modules in `config.json`, so the test will have all modules. If a customer wants to change the config she/he should also change the test accordingly. It helps prevent issues, then a person deleted another module or made changes in the config file unintentionally